datafusion/expr/src/logical_plan/plan.rs

```diff
@@ -981,14 +981,18 @@ impl LogicalPlan {
     /// // Fill in the parameter $1 with a literal 3
     /// let plan = plan.with_param_values(vec![
     ///   ScalarValue::from(3i32) // value at index 0 --> $1
-    /// ].into()).unwrap();
+    /// ]).unwrap();
     ///
     /// assert_eq!("Filter: t1.id = Int32(3)\
     ///             \n  TableScan: t1",
     ///             plan.display_indent().to_string()
     ///  );
     /// ```
```

Comment:
As above I think it would be great to extend the example to show how to handle named parameters